From 1778ebdd6802513205c29792e25ee1f472915bf3 Mon Sep 17 00:00:00 2001
From: Artur Wieczorek <artwik@wp.pl>
Date: Sun, 5 Apr 2015 09:34:11 +0200
Subject: Handle wxDateTime data in wxHandleFatalExceptions (wxMSW) only if
 wxUSE_DATETIME is set to 1.

Code using wxDateTime datatype in wxHandleFatalExceptions() should be active only if wxUSE_DATETIME == 1.

diff --git a/src/msw/main.cpp b/src/msw/main.cpp
index ad0616b..d8bb3b5 100644
--- a/src/msw/main.cpp
+++ b/src/msw/main.cpp
@@ -162,10 +162,16 @@ bool wxHandleFatalExceptions(bool doit)
         // use PID and date to make the report file name more unique
         wxString name = wxString::Format
                         (
+#if wxUSE_DATETIME
                             wxT("%s_%s_%lu.dmp"),
+#else
+                            wxT("%s_%lu.dmp"),
+#endif
                             wxTheApp ? (const wxChar*)wxTheApp->GetAppDisplayName().c_str()
                                      : wxT("wxwindows"),
+#if wxUSE_DATETIME
                             wxDateTime::Now().Format(wxT("%Y%m%dT%H%M%S")).c_str(),
+#endif
                             ::GetCurrentProcessId()
                         );
 
-- 
1.9.5.msysgit.0

